internal/commands/report_test.go

```diff
@@ -22,6 +24,13 @@ func TestReportCommand(t *testing.T) {
 	spec.Run(t, "Commands", testReportCommand, spec.Random(), spec.Report(report.Terminal{}))
 }
 
+type fakeClient struct {
```

Comment:
Do we want to make any test assertions around the contents of `pack report` w.r.t. experimental being enabled or disabled?